plugins/package-managers/spdx/src/main/kotlin/utils/SpdxResolvedDocument.kt

```diff
@@ -56,10 +56,9 @@ internal data class SpdxResolvedDocument(
     val rootDocument: ResolvedSpdxDocument,
 
     /**
-     * The name of the package manager that uses this document. This is mainly used to fill the source in generated
-     * [Issue]s.
+     * The name of the source to use when creating [Issue]s.
```

Comment:
Commit message:
> Clarify that the variable is only used to create issues, but does not
actually need to match a manager name.

Technically it's not required, but we have followed the (unwritten) convention in the past that issues created by package managers should use the manager name as source, so the explanation is a bit misleading to me. I'm fine with the change itself because this class does not need to know about this convention.

Reply:
Yes, I was not meaning to imply breaking with that convention, and have reworded the commit message(s).